### PR TITLE
Archive featured img setting

### PR DESCRIPTION
**Pull request by Misplon**
Adds a new option to the archive featured image setting to allow the removal of the archive featured image.

### CODE REVIEW

**Comment by Misplon** [comment]
@AlexGStapleton Can you please, test and review. Thanks.

File: inc/template-tags.php [hunk 1]
```diff
@@ -50,10 +50,10 @@ function vantage_author_box( $post ) { ?>
  */
 function vantage_content_nav( $nav_id ) {
 	$jetpack_infinite_scroll_active = class_exists( 'Jetpack' ) && Jetpack::is_module_active( 'infinite-scroll' );
-	//Check if we're in the Page Builder Post Loop widget.
+	// Check if we're in the Page Builder Post Loop widget.
 	$is_page_builder_post_loop_widget = class_exists( 'SiteOrigin_Panels_Widgets_PostLoop' ) &&
-	                                    method_exists( 'SiteOrigin_Panels_Widgets_PostLoop', 'is_rendering_loop' ) &&
-	                                    SiteOrigin_Panels_Widgets_PostLoop::is_rendering_loop();
+										method_exists( 'SiteOrigin_Panels_Widgets_PostLoop', 'is_rendering_loop' ) &&
```
Comment (AlexGStapleton) [inline inc/template-tags.php]: Not too sure why this was changed. Maybe I'm missing something.

Comment (Misplon) [reply inc/template-tags.php]: My bad, the spacing fixes were changes that I hadn't pushed to develop when I created this branch. I guess this spacing should be spaces not tabs?
template-tags_php_ _vantage

Comment (AlexGStapleton) [reply inc/template-tags.php]: That's likely the issue here.. Ideally, use spaces for spacing as you can always rely on one space being one space while tabs and be displayed in many different ways (2 spaces, 4 spaces, etc).

Comment (Misplon) [reply inc/template-tags.php]: Will do.

**Comment by Misplon** [comment]
Should be ok now.

**Comment by AlexGStapleton** [comment]
Looks good! 👍

**Pull request merged**